prow/spyglass/gcsartifact_fetcher.go

```diff
@@ -90,7 +90,7 @@ func newGCSJobSource(src string) (*gcsJobSource, error) {
 		source:     src,
 		linkPrefix: "gs://",
 		bucket:     gcsPath.Bucket(),
-		jobPrefix:  gcsPath.Object() + "/",
+		jobPrefix:  path.Clean(gcsPath.Object()) + "/",
```

Comment:
seems in https://github.com/kubernetes/test-infra/blob/c92564a8b88b4997f74ea476f0a2627ab8412eb1/prow/spyglass/gcsartifact_fetcher_test.go#L46 you have a test case... but the test didn't really check `exJobPrefix`？

Reply:
ah :-) https://github.com/kubernetes/test-infra/blob/c92564a8b88b4997f74ea476f0a2627ab8412eb1/prow/spyglass/gcsartifact_fetcher_test.go#L75-L80 :-)